linux/scripts/upload-to-debian.sh

```diff
@@ -101,11 +101,21 @@ if [ -n "$PUSH" ]; then
     $NOOP git push origin chore/linux/changelog
 fi
 
-git checkout -B chore/linux/cherry-pick/changelog origin/master
+if git branch -r | grep origin/beta; then
+    CLBRANCH=origin/beta
+else
+    CLBRANCH=origin/master
+fi
+
+git checkout -B chore/linux/cherry-pick/changelog ${CLBRANCH}
 git cherry-pick -x chore/linux/changelog
 if [ -n "$PUSH" ]; then
     $NOOP git push origin chore/linux/cherry-pick/changelog
 fi
 
 echo_heading "Finishing"
-git checkout master
+if git branch -r | grep origin/beta; then
```

Comment:
I'd prefer to store this as a flag earlier in the script for consistency and reuse it here.

Reply:
Done